frontend/src/utilities/useWatchGroups.tsx

```diff
@@ -23,13 +23,20 @@ export const useWatchGroups = (): {
     allowedGroups: [],
   });
 
+  const errors = React.useMemo(
+    () => ({
+      errorAdmin: groupSettings.errorAdmin,
+      errorUser: groupSettings.errorUser,
+    }),
+    [groupSettings.errorAdmin, groupSettings.errorUser],
+  );
```

Comment:
You only need to memoize because you're not breaking it out. The value of each field is the only thing that matters and those are strings. So just destruct the object, and use the two values in the Effect dependencies. No need to memo.